cmd/statusd/main.go

```diff
@@ -1,174 +1,142 @@
 package main
 
 import (
+	"flag"
 	"fmt"
+	"log"
 	"os"
-	"path/filepath"
 	"runtime"
+	"strings"
 
 	"github.com/status-im/status-go/geth/api"
 	"github.com/status-im/status-go/geth/params"
-	"gopkg.in/urfave/cli.v1"
 )
 
 var (
-	gitCommit  = "rely on linker: -ldflags -X main.GitCommit"
-	buildStamp = "rely on linker: -ldflags -X main.buildStamp"
-	app        = makeApp(gitCommit)
-	statusAPI  = api.NewStatusAPI()
+	gitCommit  = "N/A" // rely on linker: -ldflags -X main.GitCommit"
+	buildStamp = "N/A" // rely on linker: -ldflags -X main.buildStamp"
 )
 
 var (
-	// ProdModeFlag is whether we need dev or production settings
-	ProdModeFlag = cli.BoolFlag{
-		Name:  "production",
-		Usage: "Whether production settings should be loaded",
-	}
-
-	// NodeKeyFileFlag is a node key file to be used as node's private key
-	NodeKeyFileFlag = cli.StringFlag{
-		Name:  "nodekey",
-		Usage: "P2P node key file (private key)",
-	}
+	prodMode       = flag.Bool("production", false, "Whether production settings should be loaded")
+	nodeKeyFile    = flag.String("nodekey", "", "P2P node key file (private key)")
+	dataDir        = flag.String("datadir", params.DataDir, "Data directory for the databases and keystore")
+	networkID      = flag.Int("networkid", params.RopstenNetworkID, "Network identifier (integer, 1=Homestead, 3=Ropsten, 4=Rinkeby, 777=StatusChain)")
+	whisperEnabled = flag.Bool("shh", false, "SHH protocol enabled")
+	swarmEnabled   = flag.Bool("swarm", false, "Swarm protocol enabled")
+	httpEnabled    = flag.Bool("http", false, "HTTP RPC endpoint enabled (default: false)")
+	httpPort       = flag.Int("httpport", params.HTTPPort, "HTTP RPC server's listening port")
+	ipcEnabled     = flag.Bool("ipc", false, "IPC RPC endpoint enabled")
+	logLevel       = flag.String("log", "INFO", `Log level, one of: "ERROR", "WARN", "INFO", "DEBUG", and "TRACE"`)
```

Comment:
By the way, let's create a new one `loglevel` and mark the old as deprecated as well?
(in a new pr which you'll do about cluster flag:)

Reply:
@tiabc thanks! I think `log` is still ok because much shorter. And what is "cluster flag"?

Reply:
Well, let's discuss it later some time if at all. I wrote to you on slack about that long ago.